perfkitbenchmarker/providers/docker/docker_container_instance.py

```diff
@@ -152,15 +152,15 @@ def _Delete(self):
 
   @vm_util.Retry(poll_interval=10, max_retries=10)
   def _Exists(self):
-    """Returns whether the container is up and running"""
+    """Returns whether the container is up and running."""
 
-    info, returnCode = self._GetContainerInfo()
+    info, return_code = self._GetContainerInfo()
 
-    logging.info("Checking if Docker Container Exists")
-    if len(info) > 0 and returnCode == 0:
+    logging.info('Checking if Docker Container Exists')
+    if info and return_code == 0:
       status = info[0]['State']['Running']
-      if status == "True" or status is True:
-        logging.info("Docker Container %s is up and running.", self.name)
+      if status:
```

Comment:
@dPhanekham just wanted to give you a heads up about this change: I've only seen status as a bool, but let me know if you've seen otherwise (and maybe add a unit test with the mocked actual response you're seeing)